src/NUnitFramework/framework/nunit.framework.csproj

```diff
@@ -0,0 +1,64 @@
+﻿<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0">
+  <PropertyGroup>
+    <AssemblyName>nunit.framework</AssemblyName>
+    <RootNamespace>NUnit.Framework</RootNamespace>
+    <TargetFrameworks>net40;net45;netstandard1.3;netstandard1.6</TargetFrameworks>
```

Comment:
This is one of the new format `csproj` files. It is much shorter and only includes files that are outside of the project directory.